wordpress-mobile/WordPress-Android
# AppBar tweaks

This PR adds a variety of AppBar related visual changes:
 - Updated design of Site Picker on My Site screen.
 - The border around gravatar in My Site title and Pages/Posts author filter selector.
 - Updated scrolling behavior of Reader and Notifications tabs.
 - Blavatar background in Site Picker and My Site screen.

If you are interested, more details are available in the comments of this PR.

To test:
- Confirm that the site picker on My Site screen looks and behaves as expected.
- Set a site title to something long, and make sure that when it wraps to second line text size is reduced.
- Confirm that the scrolling with AppBar in Reader and Notifications behave in the same way.
- Make sure that the default site icon (globe on transparent BG) is visible over gray BG on My Site and Site Picker screen.


PR submission checklist:
